### ISSUE TITLE
Feature Request - Support for Garmin IMG maps

### ISSUE THREAD

**Issue by prdufresne** [ge0rg/aprsdroid#166]
Support for Garmin IMG map files would be very helpful.

**Comment by ge0rg** [comment]
Could you elaborate the need for those? From my impression, most garmin users actually import from OSM, and that's also what you can do in APRSdroid.

**Comment by prdufresne** [comment]
I'm aware that it's possible to import from OSM, however there are several sources of Garmin IMG maps that were generated natively in that format, including some I have generated myself by importing from Shapefile and other sources to create fully routable maps that can be loaded on Garmin devices.

**Comment by ge0rg** [comment]
Yeah, I can see how it does make sense to have those on a Garmin device. What I'm wondering about is what the benefit of the IMG maps is on an Android device, over Google Maps or OSM?

**Comment by prdufresne** [comment]
Most of us who have taken the time to create custom maps have done so because we have content layers that are not available with standard maps. For example, My maps include trails, retired roads, waterways, swamps, beaver dams and land use restrictions that are not available with Google Maps or OSM.

**Comment by Droidphilev** [comment]
Hello Ge0rg! I have to +1 this request as there are many more ready-to-go maps in this garmin .img format.  Since the future doesn't look too bright for us network radio users (No Google anymore) it would be very welcome to have this in the OSM version.

**Comment by luzemario** [comment]
+1 for this. Google coverage is good, but nothing compares to already existing IMG format maps in my country. Creating IMG maps has been done for ages, so IMG maps are much more detailed, specially outside US.
